packages/react-components/src/components/time-sync/timeSelection.tsx

```diff
@@ -52,31 +55,93 @@ const messages: ViewportMessages = {
  * parent TimeSync and affect all other viewports in
  * that TimeSync group.
  */
-export const TimeSelection = () => {
+export const TimeSelection = ({ isPaginationEnabled }: { isPaginationEnabled?: boolean }) => {
   const { viewport, setViewport } = useViewport();
+  const [disabled, setDisabled] = useState<boolean | undefined>(true);
```

Comment:
This name should be more descriptive. What is disabled? Without a more descriptive name, mental mapping is required.